docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,252 @@
+# Natural Language Processing (NLP)
+
+
+NLP is a subset of Natural Language Toolkit that specifies an interface and a protocol for basic natural language processing. Tizen enables you to use Natural Language Process (NLP) functionalities, such as language detection, parts of speech, word tokenize, lemmatize, and named entity detect   ion. For more information, see the [NLTK Forum](http://www.nltk.org/).
```

Comment:
detect   ion -> detection

@divyk Question to the review result. I think we can call language detection, word tokenize, lemmatize, and named entity detection as **NLP functionalities**.
(However I'm not sure with the words yet. tokenize? Tokenization? lemmatize? lemmatisation?)
But is it ok to say "parts of speech" as a functionality?

It is same to the below list. It is in form, "The main features of the Tizen.Nlp namespace include blahblah support."
- Word Tokenize suppport
- Part of Speech support
- Named Entity Recognizer support
- Lemmatizing support
- Language Detect support

Isn't the sentence better with "The main features of the Tizen.Nlp namespace include blahblah." form?

I'm not sure with the result here, so I hope if you can review this part again. I think asking @annie-samsung's guide also helpful to you..